README.md

```diff
@@ -1,77 +1,89 @@
 ## MCC Universal Library for Linux (uldaq)
-**Info:** Contains a library to access and control supported Measurement Computing [DAQ devices](https://www.mccdaq.com/PDFs/Manuals/Linux-hw.pdf) over the Linux platform. The UL for Linux binary name is libuldaq.
 
-**Author:** Measurement Computing
+**Info:** Contains a library to access and control supported Measurement
+Computing [DAQ
+devices](https://www.mccdaq.com/PDFs/Manuals/Linux-hw.pdf) over the
+Linux platform. The Universal Library for Linux binary name is libuldaq.
+
+**Author:** [Measurement Computing Corp (MCC)](https://libusb.info)
```

Comment:
Your link here is incorrect. Measurement Computing's website is [https://www.mccdaq.com/](https://www.mccdaq.com/).

Reply:
Good catch. Thanks.